dace/transformation/interstate/gpu_transform_sdfg.py

```diff
@@ -154,19 +160,38 @@ def can_be_applied(self, graph, expr_index, sdfg, permissive=False):
                     return False
         return True
 
-    def apply(self, _, sdfg: sd.SDFG):
+    def _output_or_input_is_marked_host(self, state, entry_node):
+        if (self.host_data is None or self.host_data == []) and (self.host_maps is None or self.host_maps == []):
+            return False
+        marked_accesses = [e.data.data for e in state.in_edges(entry_node) + state.out_edges(state.exit_node(entry_node))
```

Comment:
Wouldn't checking memlet paths and adding the src and destination data make more sense here?

Reply:
To clarify, I am considering issues similar to those identified in https://github.com/spcl/dace/pull/1708